composer.json

```diff
@@ -10,18 +10,18 @@
     }
   ],
   "require": {
-    "php": "^7.1 || ^8.0",
+    "php": "^7.2 || ^8.0",
```

Comment:
the stubs are added in 1.17.0 which only supports version from ^7.2 so we not longer can test on 7.1